tests/test_results.json

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
What is this file and where it is being used? Isn't that an accidental commit?

Reply:
I may have misunderstood the "How to contribute" instructions, there in the last step (6) it says "Attach generated test_results.json to your pull request". Is this only intended for when you have not bit exact results?